src/main/kotlin/blackjack/states/FirstTurn.kt

```diff
@@ -0,0 +1,26 @@
+package blackjack.states
+import blackjack.model.Card
+import blackjack.model.Hand
+import blackjack.view.Errors
+
+class FirstTurn(override val hand: Hand = Hand(emptyList())) : Running {
+    override fun draw(card: Card): State {
+        val hand = this.hand + card
+        if (hand.size == 2) {
+            if (hand.calculatePoints() == 21) return Blackjack(hand)
+            return Hit(hand)
+        }
+        return FirstTurn(hand)
+    }
+
+    override fun stay(): State {
+        throw IllegalStateException(Errors.INVALID_STAY.message)
+    }
+
+    override fun profit(
```

Comment:
The `profit` method looks like it could be handled commonly within the `Running` state.
That might simplify your logic!